infrastructure/iac/iac.go

```diff
@@ -408,3 +414,9 @@ func (iac *Scanner) toIssueSeverity(snykSeverity string) snyk.Severity {
 	}
 	return severity
 }
+
+func getIssueKey(affectedFilePath string, issue iacIssue) string {
+	// deepcode ignore InsecureHash: The hash isn't used for security purposes.
+	id := md5.Sum([]byte(affectedFilePath + strconv.Itoa(issue.LineNumber) + issue.PublicID))
```

Comment:
Please don't use md5. Just use sha-256. These old hash algorithms are just not needed, and the comment would be completely unnecessary.